package.json

```diff
@@ -23,20 +23,27 @@
   },
   "homepage": "https://cookbook.kirby.design",
   "scripts": {
-    "ng": "nx",
-    "nx": "nx",
     "start": "npm-run-all --parallel \"ng serve cookbook\" \"build:core:watch\"",
     "start:flows": "ng serve flows",
     "start:flows:ios": "npm run build:flows && npx cap run ios",
     "start:flows:android": "npm run build:flows && npx cap run android",
+    "start:core": "npm run build:core -- --dev --watch --serve",
+    "start:flows:prod": "ng serve flows --configuration production",
```

Comment:
Would it make more sense to switch these to lines, so `start:flows:prod` is grouped with the other `start:flows` scripts?